common/util.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+function sizeOfShape(shape) {
```

Comment:
Could you make this file as a module? And export the `sizeOfShape` and `buildConstantByNpy`?

Reply:
Please rename this file to `utils.js` that reflects it exports multiple functions.

Reply:
Can I modify the `numpy.js`? I'd like to export the `numpy` as well.

Reply:
It sounds good. numpy.js comes from https://github.com/lutzroeder/netron/blob/main/source/numpy.js. Please follow its license for derived work. https://github.com/lutzroeder/netron/blob/main/LICENSE